app/index.js

```diff
@@ -25,7 +25,7 @@ var versions = {
   'master': 'master',
   '0.4.0': 'v0.4.0',
   '0.4.1': 'v0.4.1',
-  '0.4.2': 'v0.4.2'
+  '0.4.2': '0.4.2'
```

Comment:
No, wait, that tag was published incorrectly. @lirantal the tag should in the MEAN repo should be `v0.4.2`, but instead it's `0.4.2.`